pygmt/tests/test_grd2xyz.py

```diff
@@ -0,0 +1,80 @@
+"""
+Tests for grd2xyz.
+"""
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+from pygmt import grd2xyz
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(resolution="01d", region=[-1, 1, -1, 1])
```

Comment:
```suggestion
    return load_earth_relief(resolution="01d", region=[-1, 1, 3, 5])
```
Recommend not having the lon and lat range identical for testing the columns are converted correctly